google/cloud/internal/throw_delegate.cc

```diff
@@ -16,6 +16,7 @@
 #include "google/cloud/internal/port_platform.h"
 #include <iostream>
 #include <sstream>
+#include <stdexcept>
```

Comment:
Does this fix an unrelated problem? Or is it somehow needed for the Kokoro stuff?

Reply:
Does not compile on Kokoro without it, probably because Kokoro runs gcc-4.9 and we do not test with that compiler anywhere else.  So kinda-related?  I can separate that.

Reply:
Yay for compiler fuss! >.<  This seems like the right place to fix it, so LGTM.